Makefile

```diff
@@ -117,10 +117,21 @@ init-package: ## Create the zarf init package (must `brew install coreutils` on
 release-init-package:
 	$(ZARF_BIN) package create -o build -a $(ARCH) --set AGENT_IMAGE_TAG=$(AGENT_IMAGE_TAG) --confirm .
 
+# INTERNAL used to build the dos games packages for release
+build-release-packages:
+	$(ZARF_BIN) package create -o build -a amd64 examples/dos-game --confirm
+	$(ZARF_BIN) package create -o build -a arm64 examples/dos-game --confirm
+
+
+# INTERNAL used to publish the dos games packages to GHCR
+publish-release-packages:
+	$(ZARF_BIN) package publish ./build/zarf-package-dos-games-amd64-0.0.1.tar.zst oci://ghcr.io/defenseunicorns/zarf-package/dos-games
+	$(ZARF_BIN) package publish ./build/zarf-package-dos-games-arm64-0.0.1.tar.zst oci://ghcr.io/defenseunicorns/zarf-package/dos-games
+
 build-examples: ## Build all of the example packages
 	@test -s $(ZARF_BIN) || $(MAKE) build-cli
 
-	@test -s ./build/zarf-package-dos-games-$(ARCH).tar.zst || $(ZARF_BIN) package create examples/dos-games -o build -a $(ARCH) --confirm
+	@test -s ./build/zarf-package-dos-games-$(ARCH)-0.0.1.tar.zst || $(ZARF_BIN) package create examples/dos-games -o build -a $(ARCH) --confirm
```

Comment:
Can we change this to 1.0.0 to make it seem less experimental (since this is our premier example)